docs/flashbots-auction/miners/mev-geth-spec/v05-rpc.mdx

```diff
@@ -0,0 +1,209 @@
+}
+```
+
+## eth_sendMegabundle
+
+### Description
+
+Sends a megabundle to the miner. The megabundle has to be executed at the beginning of the block (before any other transactions), with bundle transactions executed exactly in the same order as provided in the bundle. Can only be called by a relay listed in the `miner.trustedrelays` config.
+
+| Name | Type | Description | Comment
+--------|----------|-----------|-----------
+txs |	`Array<Data>` | Array of signed transactions (`eth_sendRawTransaction` style, signed and RLP-encoded)	| a no-op in the light mode
+blockNumber	|`Quantity`	|Exact block number at which the bundle can be included.	|bundle is evicted after the block
+minTimestamp	|`Quantity`	|Minimum (inclusive) block timestamp at which the bundle can be included. If this value is 0 then any timestamp is acceptable.
+maxTimestamp	|`Quantity`	|Maximum (inclusive) block timestamp at which the bundle can be included. If this value is 0 then any timestamp is acceptable.
+revertingTxHashes	|Array<`Data`>	|Array of tx hashes within the bundle that are allowed to cause the EVM execution to revert without preventing the bundle inclusion in a block.
+relaySignature	|Array<`Data`>	|An secp256k1 signature signed with an address from the `miner.trustedrelays`. Message signed is a Keccak hash of RLP serialized sequence that contains the following items: array of txs (a sequence of byte arrays representing RLP serialized txs); minTimestamp serialized as an int256, like in the devp2p specification; maxTimestamp serialized as an int256, like in the devp2p specification; revertingTxHashes serialized as an array of byte arrays.
+### Returns
+
+{`boolean`} - `true` if megabundle has been accepted by the node, otherwise `false`
+
+### Example
+
+```bash
+# Request
+curl -X POST -H 'Content-Type: application/json' --data '{
+    "id": 1337,
+    "jsonrpc": "2.0",
+    "method": "eth_sendMegabundle",
+    "params": [
+        {
+          "txs" : [
+              "0x02f8b30181b18502cb417800853a352944008307a12094b893a8049f250b57efa8c62d51527a22404d7c9a80b844095ea7b300000000000000000000000093e17e368e82dd24bed931091f831b5bed3f711effffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffc080a027148354c23bb016147ed68014b2aa13c43a4feef36274be88ef58d25f91e20fa05ccc423d4e9e1de88515adf3245df69db8c05b1ce345a738c75b06c87a96f878",
+              "0x02f8b30181b28502cb417800853a352944008307a12094d5281bb2d1ee94866b03a0fccdd4e900c8cb509180b844095ea7b300000000000000000000000093e17e368e82dd24bed931091f831b5bed3f711effffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffc001a042f32acc8631c8c1c81d3d77a637cdab162477077d5041285ea7d73313d6b02ca07b18b432f4921795f84916b3c9398e2fd65580e372b9fa7e018e543ffbc00375"
+          ],
+          "blockNumber" : "0xce7b22",
+          "minTimestamp" : "0x0",
+          "minTimestamp" :"0x0",
+          "revertingTxHashes": [],
+          "relaySignature" :[update docs with a valid sig so people can use it for testing],
```

Comment:
Did we ever update the docs with a valid megabundle relay signature?

Reply:
no, asked on Discord now